app/new_service_manifest.yml

```diff
@@ -36,7 +36,15 @@
     - educationPricing
     - trialOption
     - freeOption
+-
+  name: Pricing document
```

Comment:
Every time I see that acronym, I think of the calling card my mom uses to phone overseas: the [SIFA card](http://www.sifacallingcard.com/).